src/Umbraco.Web.UI.Client/src/common/directives/components/forms/umbfocuslock.directive.js

```diff
@@ -149,6 +149,12 @@
                 // Whenever the DOM changes ensure the list of focused elements is updated
                 function domChange() {
                     getFocusableElements();
+
+                    // When an element is remove or become not focusable, but focus is on it,
+                    // we need to ensure a now focus inside the trap, else the focus can escape the trap.
+                    if (focusableElements.filter(elm => elm == document.activeElement).length === 0) {
```

Comment:
Could it use the following instead?

```
if(focusableElements.some(elm => elm == document.activeElement)) {
```

Reply:
@bjarnef I have made a slighty change to the line, instead of filter or some. I used the IndexOf()
IndexOf is the most readable line and fasted execution time, https://www.measurethat.net/Benchmarks/Show/20458/0/some-vs-filter-vs-indexof-vs-includes1123123123

Reply:
Okay, I ran the test and it says includes are the fastest 😄

![Screenshot_20220817-133436_Outlook](https://user-images.githubusercontent.com/2919859/185108946-b22b46cc-6f70-4efe-b085-db477559fc98.jpg)